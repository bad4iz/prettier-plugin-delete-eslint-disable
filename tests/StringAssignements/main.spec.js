const { format } = require('prettier')

/**
 * @param {string} code
 * @param {prettier.Options} [options]
 */
const prettify = (code, options) =>
    format(code, { plugins: ['.'], filepath: 'file.ts' })

describe('file main.spec', () => {
    it('should ', () => {
        expect.hasAssertions()
        // Arrange (всякие моки)

        // Act
        const res = prettify(`
        /* eslint-disable */
        lodash ( )`)
        // Assert
        expect(res).toMatchSnapshot()
    })
})
