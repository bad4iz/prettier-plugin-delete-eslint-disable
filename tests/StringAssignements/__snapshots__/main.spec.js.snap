// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`file main.spec should  1`] = `
lodash();
"
`;
